Components/ListSearchResults/Partials/Results/_style.scss

```diff
@@ -48,9 +54,33 @@
     }
   }
 
-  .excerpt {
-    color: #9b9b9b;
+}
+
+.post {
+  &-title {
```

Comment:
I feel this should rather be a default heading selector with default styles (like a `h3` tag). We might need to review the overall font-sizes of this component then, but I think it's better to keep it generic than making a lot of custom typographic changes just for this component.